C# Examples/EposServiceConsumer/README.md

```diff
@@ -0,0 +1,92 @@
+|`EPOS_API_SANDBOX_TABLE_1_ID`|A table id for your active area. Please take not of the number of covers of this table.|
+|`EPOS_API_SANDBOX_TABLE_6_ID`|A table id for your active area.|
+|`EPOS_API_SANDBOX_TABLE_7_ID`|A table id for your active area. Please take not of the number of covers of this table.|
+|`EPOS_API_SANDBOX_API_TEST_CHANNEL_ID`|The id for one of your channels. ONLINE channel by defaults is 3|
+|`EPOS_API_SANDBOX_FOREVER_PROMOTION_ID`|The id for one of your active promotions.|
+
+<br/>
+<br/>
+
+|Variables for Testing EPOS API WITH BUMPING |Description|
+| --- | --- |
+|`EPOS_API_SANDBOX_WITH_BUMPING_CONSUMER_SECOND_SECRET`|he second secret for the restaurant you are trying to access.Varies per restaurant|
+|`EPOS_API_SANDBOX_WITH_BUMPING_RESTAURANT_ID`|The Id of the restaurant you are trying to access.|
+|`EPOS_API_SANDBOX_WITH_BUMPING_SEGMENT_ID`|The id of the service that is currently active for your restaurants|
+|`EPOS_API_SANDBOX_WITH_BUMPING_SEGMENT_NAME`|The name of the service that is currently active in your restaurant||
+|`EPOS_API_SANDBOX_WITH_BUMPING_RESTAURANT_AREA_ID`|The area id of an area in your current active segement|
+|`EPOS_API_SANDBOX_WITH_BUMPING_TABLE_1_ID`|A table id for your active area. Please take not of the number of covers of this table.|
+|`EPOS_API_SANDBOX_WITH_BUMPING_TABLE_6_ID`|A table id for your active area.|
+|`EPOS_API_SANDBOX_WITH_BUMPING_TABLE_7_ID`|A table id for your active area. Please take not of the number of covers of this table.||
+|`EPOS_API_SANDBOX_WITH_BUMPING_API_TEST_CHANNEL_ID`|The id for one of your channels. ONLINE channel by defaults is 3|
+|`EPOS_API_SANDBOX_WITH_BUMPING_FOREVER_PROMOTION_ID`|The id for one of your active promotions.|
+<br/>
+<br/>
+
+| Additional Variables for Unit Testing      |Description|
+| --- | --- |
+|`Restaurant_Name`| The name of your restaurant as it appears under Settings->Details->MicrositeName|
+|`Table_Join_Covers`|The sum of covers for table ids 1+7|
+ 
+ 
+ 
+ # Steps on how to retrieve the variables
+ 
+ ## Authorization
+ 
+ The Consumer Key and Consumer Secret identify which API account is making the request, and the Second Secret identifies the restaurant you are accessing. This makes sure that you aren't able to make requests to restaurants you don't have access to.
+
+The Consumer Key and Consumer Secret are standard parts of OAuth, but the Second Secret and Scope are specific to our implementation. 
+
+The above variables should be provided to you by default and are all requried for gaining access to the rest of the API.
+
+### Step 1:  Get a Request Token
+
+END POINT:https://app.rdbranch.com/OAuth/V10a?second_secret={{RDEposSecondSecret}}&scope=http://app.restaurantdiary.com/WebServices/Epos/v1
+
+Authorization Parameters:  `EPOS_CONSUMER_KEY` and  `EPOS_CONSUMER_SECRET`
+
+Making this API request will result in generating a request token that will be used in Step 2 to create an Access Token.
```

Comment:
Is it worth showing a code example of how to do this with the example code? Otherwise it's a bit confusing what this is trying to tell you.